BPOR/NIHR.GovUk.AspNetCore.Mvc/GovUkDate.cs

```diff
@@ -5,6 +5,15 @@ namespace NIHR.GovUk.AspNetCore.Mvc;
 public class GovUkDate : IValidatableObject
 {
 
+    private readonly int _minYear;
+    private readonly int _maxYear;
+
+    public GovUkDate(int minYear, int maxYear)
```

Comment:
This should be handled through attributes on the date property of the parent model.